ibm/service/scc/resource_ibm_scc_profile_attachment.go

```diff
@@ -109,7 +109,7 @@ func ResourceIbmSccProfileAttachment() *schema.Resource {
 			},
 			"notifications": {
 				Type:        schema.TypeList,
-				Required:    true,
+				Optional:    true,
```

Comment:
Can we add MaxItems to 1 because I see we were accessing only .0 position element

Reply:
Added the maxItems property.
Testcase :
<img width="793" alt="image" src="https://github.com/IBM-Cloud/terraform-provider-ibm/assets/33832784/6a9c7ff3-2933-4a39-b796-b7bfb5e3c9bf">